content/en/user-guide/aws/secretsmanager/index.md

```diff
@@ -0,0 +1,173 @@
+    "Name": "test-secret",
+    "Description": "LocalStack Secret",
+    "LastChangedDate": 1692882479.857329,
+    "VersionIdsToStages": {
+        "a50c6752-3343-4eb0-acf3-35c74f00f707": [
+            "AWSCURRENT"
+        ]
+    },
+    "CreatedDate": 1692882479.857329
+}
+```
+
+You can also get a list of the secrets available in your local environment that have **Secret** in the name using the [`ListSecrets`](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_ListSecrets.html) API. Execute the following command:
+
+{{ <command> }}
+$ awslocal secretsmanager list-secrets \
+    --filters Key=name,Values=Secret
+{{ </command> }}
+
+### Get the secret value
+
+To retrieve the value of the secret you created earlier, you can use the [`GetSecretValue`](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html) API. Execute the following command:
+
+{{ <command> }}
+$ awslocal secretsmanager get-secret-value \
+    --secret-id test-secret
+{{ </command> }}
+
+The following output would be retrieved:
+ 
+```bash
+{
+    "ARN": "arn:aws:secretsmanager:us-east-1:000000000000:secret:test-secret-pyfjVP",
+    "Name": "test-secret",
+    "VersionId": "a50c6752-3343-4eb0-acf3-35c74f00f707",
+    "SecretString": "{\n  \"username\": \"admin\",\n  \"password\": \"password\"\n}\n",
+    "VersionStages": [
+        "AWSCURRENT"
+    ],
+    "CreatedDate": 1692882479.857329
+}
+```
+
+You can tag your secret using the [`TagResource`](https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_TagResource.html) API. Execute the following command:
+
+{{ <command> }}
+$ awslocal secretsmanager tag-resource \
+    --secret-id test-secret \
+    --tags Key=Environment,Value=Development
+{{ </command> }}
+
+### Rotate the secret
+
+To rotate a secret, you need a Lambda function that can rotate the secret. You can copy the code from a [Secrets Manager template](https://docs.aws.amazon.com/secretsmanager/latest/userguide/reference_available-rotation-templates.html) or you can use a [generic Lambda function](https://github.com/aws-samples/aws-secrets-manager-rotation-lambdas/blob/master/SecretsManagerRotationTemplate/lambda_function.py) that rotates the secret.
+
+Zip the Lambda function and create a Lambda function using the [`CreateFunction`](https://docs.aws.amazon.com/lambda/latest/dg/API_CreateFunction.html) API. Execute the following command:
+
+{{ <command> }}
+$ zip my-function.zip lambda_function.py
+$ aws lambda create-function \
+    --function-name my-rotation-function \
+    --runtime python3.9 \
+    --zip-file fileb://my-function.zip \
+    --handler my-handler \
+    --role arn:aws:iam::000000000000:role/service-role/rotation-lambda-role
+{{ </command> }}
+
+You can now set a resource policy on the Lambda function to allow Secrets Manager to invoke it using [`AddPermission`](https://docs.aws.amazon.com/lambda/latest/dg/API_AddPermission.html) API. Execute the following command:
```

Comment:
```suggestion
You can now set a resource policy on the Lambda function to allow Secrets Manager to invoke it using [`AddPermission`](https://docs.aws.amazon.com/lambda/latest/dg/API_AddPermission.html) API.

Please note that this is not required with the default LocalStack settings, since IAM permission enforcement is disabled by default.

Execute the following command:
```

Reply:
This is actually only required with IAM enforcement enabled. I don't think we usually highlight these cases?